environments/production/postgresql.yml

```diff
@@ -8,7 +8,7 @@ postgres_override:
 
 pgbouncer_override:
   pgbouncer_default_pool: 290
-  pgbouncer_max_connections: 3000
+  pgbouncer_max_connections: 10000
```

Comment:
just curious, what was your reasoning for tripling the pgbouncer_max_connections vs the 4x improvement for `instance_type`?

Reply:
There's no super data-driven reason for it. I tend to raise limits exponentially (i.e. by a percentage rather than a fixed amount), and tend to follow round number near-exponential patterns like (1, 2, 5, 10, 20, 50, 100, 200, 500, 1000...) or (1, 3, 10, 30, 100, 300, 10000). So I saw 3,000 and just went to the next number in the 1-3-10 pattern. Didn't give it much more thought than that to be honest